src/webgl/p5.RendererGL.js

```diff
@@ -631,7 +635,22 @@ p5.RendererGL.prototype._getColorShader = function () {
   return this._defaultColorShader;
 };
 
+p5.RendererGL.prototype.getTexture = function (img) {
+  // efficiency?
+  var f = function(element) {
```

Comment:
Curious why you don't use a map here for simplicity.

```js
if (this.textures[img] === undefined) {
  this.textures[img] = new p5.Texture(this, img);
}
return this.textures[img];
```

Performance of finding a single element in the array is good, but will probably degrade as the list grows (though there is a hardware limit to how many textures you could have). Map lookup times for the single element are comparable.